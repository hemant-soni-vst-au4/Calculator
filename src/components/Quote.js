import React from 'react';
import Navbar from './Navbar';

export default function Aboutus() {
  return (
    <div>
      <h1>Quote of day</h1>
    </div>
  );
}
