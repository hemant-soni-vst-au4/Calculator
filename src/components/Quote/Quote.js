import React from 'react';

export default function Quote() {
  return (
    <div>
      <h1>Quote of day</h1>
    </div>
  );
}
