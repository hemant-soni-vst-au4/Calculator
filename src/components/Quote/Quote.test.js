import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from './Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

describe('Quote', () => {
  test('renders Quote component', () => {
    render(<Quote />);
    expect(screen.getByText('Quote of day')).toBeInTheDocument();
  });
});
