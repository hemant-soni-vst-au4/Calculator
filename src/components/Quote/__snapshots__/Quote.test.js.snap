// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Quote of day
  </h1>
</div>
`;
